File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/RolePlayer/RolePlayerBenefit.cs
using RMA.Service.ClientCare.Contracts.Entities.Product;

namespace RMA.Service.ClientCare.Contracts.Entities.RolePlayer
{
    public class RolePlayerBenefit : Benefit
    {
        public string RolePlayerName { get; set; }
        public string BenefitName { get; set; }
        public string ProductOptionName { get; set; }
        public int Age { get; set; }
        public bool AgeIsYears { get; set; }
        public bool IsStatedBenefit { get; set; }
        public bool Selected { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/role-player-benefit.ts
import { Benefit } from '../../../product-manager/models/benefit';

export class RolePlayerBenefit extends Benefit {
  rolePlayerName: string;
  memberType: string;
  age: number;
  ageIsYears: boolean;
  benefitName: string;
  productOptionName: string;
  isStatedBenefit: boolean;
}
